src/component/tooltip/TooltipView.js

```diff
@@ -586,7 +610,8 @@ export default echarts.extendComponentView({
                 }
             }, this);
             this._ticket = asyncTicket;
-            html = formatter(params, asyncTicket, callback);
+            html = cacheHtml || formatter(params, asyncTicket, callback);
+            this._setTooltipCache(key, html);
```

Comment:
If `triggerOnce` is not enabled, `enableCache` will be false, so there is no need to call `_setTooltipCache`.
We should wrapped this code line with `if`.

Reply:
Maybe no need to wrap, I saw that it won't invoke if `key` is not specified in function `_setTooltipCache`.

Reply:
From a semantic perspective, it is better to wrap